addons/payment/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,31 @@
+---Fields in module 'payment'---
+payment      / account.invoice          / transaction_ids (many2many)   : NEW relation: payment.transaction
```

Comment:
I'm not able to find where is this field populated. Do you? If there's a flow that feeds it, then we should try to feed past invoices/transactions the same way.

Reply:
The `many2many` is filled in https://github.com/OCA/OpenUpgrade/blob/12.0/addons/payment/models/account_invoice.py#L81 and in https://github.com/OCA/OpenUpgrade/blob/12.0/addons/payment/models/payment_acquirer.py#L633. I don't know if I should do something about. Could you check?

Reply:
I have finally filled this from linked account.payment